src/auth.interceptor.ts

```diff
@@ -7,16 +7,9 @@ import {
   HttpRequest,
   HttpErrorResponse
 } from '@angular/common/http';
-import { Subject } from 'rxjs/Subject';
-import { Observable } from 'rxjs/Observable';
 
-import {
-  map,
-  first,
-  switchMap,
-  _throw,
-  _catch
-} from './rxjs.util';
+import { Observable, Subject, throwError } from 'rxjs';
```

Comment:
could you use explicit imports from rxjs to not to import everything from rxjs?

Reply:
One of the changes of rxjs 6 is that now there are no explicit imports, everything is imported from `rxjs`,` rxjs / operators`, etc. They cleaned the surface of the api in that version.